src/torch/README.md

```diff
@@ -0,0 +1,178 @@
+# Notes for Design Discussion
+
+
+## File structure
+
+Currently the source is found here:
+```md
+SIRF
+├── src
+│   ├── torch
+│       ├── CMakeLists.txt
+│       ├── README.md
+│       └── SIRF_torch.py
+```
+Currently installed in:
+```md
+INSTALL
+├── python
+│   ├── sirf
+│       └── SIRF_torch.py
+```
+
+The classes are accessed as `sirf.SIRF_torch.TheClass`.
```

Comment:
I'd prefer `sirf.torch.TheClass`, which presumably could be done by having `__init__.py` importing everything from `SIRF_torch`.